ZUGFeRD/Profile.cs

```diff
@@ -73,7 +73,12 @@ public enum Profile
         /// Invoice format based on EU Directive 2014/55/EU, adopted to Germany in E-Invoice Law of April 4, 2017 (BGBl. I p. 770).
         /// Important note: using this profile will generate a version 1 XRechnung (as valid until 31/12/2020)
         /// </summary>
-        XRechnung1 = 64
+        XRechnung1 = 64,
+
+        /// <summary>
+        /// The e-reporting 

```

Comment:
Can you please add more documentation here? You may use deepl.com or chatGPT for translation if you like.

Reply:
done

Reply:
Looks good, @Athilla . Now it's just the writer missing along with the corresponding tests!